tests/unit/test_views.py

```diff
@@ -118,7 +118,7 @@ def test_get_srpm_logs(client):
 
 
 def test_system_api(client):
-    response = client.get("/api/system")
+    response = client.get("/system")
```

Comment:
Does this work?

Reply:
yes, since we set the `/api/` prefix on the FastAPI level, therefore I needed to change the test